### PR TITLE
Crits

### PR DESCRIPTION
**Pull request by ITarako**

### CODE REVIEW

File: frontend/src/components/kl-add-item.js [hunk 1]
```diff
@@ -10,16 +10,23 @@ export default {
             focused: false,
             typeList: transTypeList(Item.types),
             type: Item.types[0],
-            minDmg: 2,
-            maxDmg: 3,
+            minDmg: 1,
+            maxDmg: 2,
+            critChance: 0,
+            critDmg: 0
```
Comment (Eoksni) [inline frontend/src/components/kl-add-item.js]: group these into object, they pollute the namespace

File: frontend/src/i18ns/ru.js [hunk 2]
```diff
@@ -6,6 +6,8 @@ export default {
         minDmg: "Минимальный урон",
         maxDmg: "Максимальный урон",
         dps: "Урон в секунду",
+        critChance: "Вероятность критического удара",
```
Comment (Eoksni) [inline frontend/src/i18ns/ru.js]: I think "Шанс" is better suited here.

File: domain/src/Item.js [hunk 3]
```diff
@@ -21,9 +21,11 @@ function calcAps(item) {
 
 function calcDps(item) {
     let res = (((item.minDmg + item.maxDmg) / 2) * item.aps);
-    return (item.critChance && item.critDmg)
-        ? (res + (item.critChance / 100) * (item.critDmg / 100))
-        : res;
+    if (item.critChance && item.critDmg) {
+        if (item.critChance > 100) item.critChance = 100;
```
Comment (Eoksni) [inline domain/src/Item.js]: are you sure that is how crit chance about 100 is treated in diablo?

Comment (ITarako) [inline domain/src/Item.js]: nope х)

Comment (Eoksni) [inline domain/src/Item.js]: for future reference - if you are not sure how to implement a certain feature, you must ask the client (me in this case) or at least put a comment in the code with your thoughts. Like so:
```js
        // HACK: i'm not sure if it is possible to reach more than 100% crit 
        // chance in diablo and how it handles it so I just hard limit it here.
        if (item.critChance > 100) item.critChance = 100;
 
```

Comment (Eoksni) [inline domain/src/Item.js]: and don't silently ignore my comments :angry: i told you about tests for calcDps and you neither added them nor responded with anything.

Comment (ITarako) [inline domain/src/Item.js]: after these commits, i began write module `Item.test.js`. but i saw that you created исус for calcDps test & thought that everything was ok

Comment (Eoksni) [inline domain/src/Item.js]: well how was I supposed to know about that ? don't hesitate to drop short comments about your intentions

**Pull request merged**